extensions/ql-vscode/src/remote-queries/view/HorizontalSpace.tsx

```diff
@@ -1,7 +1,9 @@
-import * as React from 'react';
+import styled from 'styled-components';
 
-const HorizontalSpace = () => (
-  <span className="vscode-codeql__horizontal-space" />
-);
+const HorizontalSpace = styled.div<{ size: 1 | 2 | 3 }>`
```

Comment:
I see this is a start - we'll probably want to play around with this sizes and see what feels right.

Reply:
That's neat. I've never seen this before.